### DISCUSSION TITLE
Relation not found error for postgresql

### DISCUSSION THREAD

**Discussion by GarryXian** [Budibase/budibase#5919] [Help]
![image](https://user-images.githubusercontent.com/44110688/168740829-15d54005-a3ae-46f1-8b5d-78081218f9a7.png)
Hi,

We encountered a error - 'relation not exist' ,  but the table actually exist in the DB, and Budibase has fetched schema and tables sucessfully.

**Comment by melohagan** [comment]
Hi @GarryXian 

What does the relationship look like?

**Reply by GarryXian** [reply]
Hi,

We didn't set up any relationships in Buidibase, 
![image](https://user-images.githubusercontent.com/44110688/169737780-d87fdd7d-e63e-4f81-8cdb-e547b37bdc16.png)
and you can refer to the DDL for the table
![image](https://user-images.githubusercontent.com/44110688/169737905-74c09d72-1430-4884-910f-8072185917eb.png)

**Reply by melohagan** [reply]
Thank you for the info. 

If you deleted the PostgreSQL connection, and re-added and re-fetched, do you still see the same error? 

Are you self hosted or cloud? What version of Budibase is your app using?

**Reply by GarryXian** [reply]
Hi,

we are self hosted, and  we are using 1.0.158 version.

And we have tried to delete and re-add the data source, the same error still happens.

And we roll back to use version 1.0.99, the errors doesn't happend any more, so may it be a bug for version 1.0.158?

**Reply by melohagan** [reply]
Brilliant, that's good to know. Do any of the other tables that are working fine have any of the following column types?

- timestamp with time zone
- json
- nextval('io_id_seq')

If there's an issue with column types, then most likely the last one is problematic . If you tried using a basic int primary key on the io table does it work in 1.0.158?

**Comment by GarryXian** [comment]
我已收到您的邮件会尽快阅读和回复
